sbin/rosa-svn-pre-commit

```diff
@@ -26,4 +26,4 @@
 # DESCRIPTION
 #     Ensure that an SVN commit conforms to the rules of Rosie.
 #-------------------------------------------------------------------------------
-exec python2 -m rosie.svn_pre_commit "$@"
+exec python3 -m rosie.svn_pre_commit "$@"
```

Comment:
This works locally when I explicitly set `some/path/on/my/system/to/a/python3` but fails when I use python3 alone citing: `/anon/home/tim/rose.git/sbin/rosa-svn-pre-commit: line 29: exec: python3: not found`. I presume that this is an effect of being in the sbin folder rather than the bin folder. Thoughts welcomed.